docs/Using-Fleet/Permissions.md

```diff
@@ -51,7 +51,7 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | Run any query as [live query](https://fleetdm.com/docs/using-fleet/fleet-ui#run-a-query) against all hosts                                 |          | ✅         | ✅         | ✅    |         |
 | Create, edit, and delete queries                                                                                                           |          |            | ✅         | ✅    | ✅      |
 | View all queries\**                                                                                                                        | ✅       | ✅         | ✅         | ✅    |         |
-| Add, edit, and remove queries from all schedules                                                                                           |          |            | ✅         | ✅    | ✅      |
+| Manage [query automations](https://fleetdm.com/docs/using-fleet/fleet-ui#schedule-a-query)                                                 |          |            |           | ✅    |       |
```

Comment:
I think maintainer and GitOps users should be able to manage query automations— is that not the case? (Just checked the main issue & didn't specify)